File: 07_문자열/김형민/1152.py
문자 = input()
답 = 1
for i in range(len(문자)):
    if len(문자) == 1 :
        if 문자[i] == " ":
            답 = 0
    if 문자[i] == " ":
        if i == 0:
            답 = 답 - 1
        elif len(문자)-1 == i:
            답 = 답 - 1
        
        답 = 답 + 1        
print(답)

File: 07_문자열/이정현/1152.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString();

let wordsArr = input.trim().split(' ');
let countOfWord = 0;

for (let i = 0; i < wordsArr.length; i++) {
    if (wordsArr[i] !== '') {
        countOfWord++;
    }
}

console.log(countOfWord);